src/Tabs.tsx

```diff
@@ -1 +1,22 @@
 // implement a component here
+import React from 'react';
+
+interface Props {
+  tabs: Tab[],
+  onTabSelect: (tabId: string) => void,
+}
+
+export const Tabs: React.FC<Props> = ({ tabs, onTabSelect }) => {
+  return (
+    <>
+      {tabs.map((tab: Tab) => (
+        <button
+          type="button"
+          onClick={() => onTabSelect(tab.id)}
```

Comment:
Will be better to call this function only if tab id is now equal to selected tab id